src/app/services/events.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { StompService } from 'ng2-stomp-service';
+import { environment } from '../../environments/environment';
+import { Event } from '../models/event';
+import { EventType } from '../models/eventType';
+
+@Injectable()
+export class EventsService {
+
+  constructor(private stomp: StompService) {
+    // Configure stomp client
+    stomp.configure({
+      host: environment.sf_server_address + '/events',
+      debug: environment.stomp_debug,
+      queue: { 'init': false, 'events': false }
+    });
+
+    // Start socket connection
+    stomp.startConnect().then(() => {
+      stomp.done('init');
+      console.log('Da client has konnected');
```

Comment:
really? :smile:  we aint time for that, mejor borremos ese console.log

Reply:
hahahaha